s3torchconnector/tst/e2e/test_e2e_s3_lightning_checkpoint.py

```diff
@@ -0,0 +1,217 @@
+#  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#  // SPDX-License-Identifier: BSD
+import pytest
+import random
+import torch
+
+from pathlib import Path
+from typing import Dict, Any
+from lightning.pytorch.callbacks import ModelCheckpoint
```

Comment:
We may need to add dependency on https://pypi.org/project/pytorch-lightning/ to have access to `ModelCheckpoint`